question/bank/managecategories/classes/question_category_object.php

```diff
@@ -140,22 +154,34 @@ public function initialize($page, $contexts, $currentcat, $defaultcategory, $tod
         if (!$currentcat) {
             $this->catform->set_data(['parent' => $defaultcategory]);
         }
+        // Checkbox Form.
+        $checked = get_user_preferences('question_bank_qbshowdescr');
```

Comment:
just a thought, shouldnt it be a bit more specific? naming seems related to qbank base where is related to category description, can be named something better if its not too much work

Reply:
name changed to `qbank_managecategories_showdescr`